src/sparse/impl/KokkosSparse_spgemm_jacobi_sparseacc_impl.hpp

```diff
@@ -0,0 +1,1542 @@
+/*
+//@HEADER
```

Comment:
I was looking for a case where d_inv will be read once for the entire team when all threads in the team are working on a row, couldn't see it. May be you haven't considered that level of optimization.

Reply:
I think this line is already doing what you ask for: https://github.com/seheracer/kokkos-kernels/blob/247a195060fc1e167e38a8a62ce3a7f34c2515c9/src/sparse/impl/KokkosSparse_spgemm_jacobi_sparseacc_impl.hpp#L462

For all threads who work on the row with id row_index, dinv(row_index,0) is read once. No?